Source/LogWriter.swift

```diff
@@ -101,7 +104,7 @@ open class ConsoleWriter: LogModifierWriter {
 
         switch method {
         case .print: print(message)
-        case .nslog: NSLog("%@", message)
+        case .nslog: message.withCString { NSLog("%s", $0) }
```

Comment:
At first glance this seems like it should work (docs for `withCString` say it yields a UTF-8 string) , but when I run the iOS Sample app with the `ConsoleWriter` set to `.nslog`, the unicode chars in the sample messages show up in the Xcode console as garbage. 

The existing code does not have this problem. More investigation is needed, but I'm wondering if the `%s` formatter implementation in `NSLog` only does ASCII...

Reply:
New proposition, it is possible to use NSLog and pass directly the string without the formatter. Works on linux, and tested with the app configured in NSLog, unicode was fine.

Reply:
Bad practice because the first argument to `NSLog` is assumed to be a formatter string. If the string was passed directly and had any formatter characters in it you would have unexpected results or a crash.

Reply:
Might need to just #if on the NSLog to have two different versions: one for Linux and one for the Apple OS's.

Reply:
Okay sorry I'm quite new to swift. I will look into that